eng/scripts/Set-VcpkgWriteModeCache.ps1

```diff
@@ -1,6 +1,7 @@
 param(
     [string] $StorageAccountKey
 )
+Uninstall-AzureRm
```

Comment:
Why did we need this? Perhap add info about it in the issue you created about having to manually Install-Module as well.

Reply:
When installing `Az.Storage` @ 4.3.0 we get a warning message recommending running `Uninstall-AzureRm`. This was intended to eliminate that warning. 

However... we were able to fix this by running `Uninstall-AzureRm` and then using the module helpers.